libs/template/presets/fulfillment/experience/pages/login-page.ts

```diff
@@ -0,0 +1,37 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+import { Size } from '@spryker-oryx/utilities';
+
+export const fulfillmentLoginPage: ExperienceComponent = {
+  id: 'fulfillment-login-page',
+  type: 'Page',
+  meta: {
+    title: 'Login Page',
+    route: '/login',
+    description: 'Login Page Description',
+  },
+  options: {
+    rules: [
+      { layout: 'flex', justify: 'center' },
+      {
+        query: { breakpoint: Size.Sm },
+        width: '100%',
+      },
+      {
+        query: { breakpoint: Size.Md },
+        width: '414px',
+      },
+    ],
+  },
```

Comment:
we should not need to repeat the width here, the container size should already provide this. Happy to pair program on this one if needed.